src/scales/scale.time.js

```diff
@@ -676,10 +677,10 @@ module.exports = Scale.extend({
 		var minorFormat = formats[me._unit];
 		var majorUnit = me._majorUnit;
 		var majorFormat = formats[majorUnit];
-		var majorTime = +adapter.startOf(time, majorUnit);
+		var isMajorTime = isNaN(index) ? false : ticks[index].major;
```

Comment:
Do we need to verify that `ticks[index]` exists?

Reply:
I'm not really sure it's necessary, but it doesn't hurt, so I've gone ahead and changed it